sormas-backend/src/main/resources/META-INF/persistence.xml

```diff
@@ -87,23 +87,25 @@
 		<class>de.symeda.sormas.backend.externalmessage.labmessage.SampleReport</class>
 
 
-
-
 		<exclude-unlisted-classes>true</exclude-unlisted-classes>
 
 		<shared-cache-mode>ENABLE_SELECTIVE</shared-cache-mode>
 		<validation-mode>CALLBACK</validation-mode>
-		
-		 <properties>
-         	<property name="hibernate.dialect" value="de.symeda.sormas.backend.ExtendedPostgreSQL94Dialect"/>
-			<property name="hibernate.criteria.literal_handling_mode" value="bind" />
-         	<property name="hibernate.transaction.jta.platform" value="org.hibernate.service.jta.platform.internal.SunOneJtaPlatform"/>
-			<property name="hibernate.show_sql" value="false" />
-			<property name="hibernate.format_sql" value="false" />
-         	<!-- don't validate on release, otherwise the automatic db update won't work
-         	<property name="hibernate.hbm2ddl.auto" value="validate"/>
-         	 -->
-      	</properties>
+
+		<properties>
+			<property name="hibernate.dialect" value="de.symeda.sormas.backend.ExtendedPostgreSQL94Dialect"/>
+			<property name="hibernate.criteria.literal_handling_mode" value="bind"/>
+			<property name="hibernate.transaction.jta.platform"
+					  value="org.hibernate.service.jta.platform.internal.SunOneJtaPlatform"/>
+			<property name="hibernate.show_sql" value="false"/>
+			<property name="hibernate.format_sql" value="false"/>
+
+			<property name="hibernate.cache.use_second_level_cache" value="true"/>
+			<property name="hibernate.cache.region.factory_class" value="org.hibernate.cache.ehcache.internal.SingletonEhcacheRegionFactory"/>
+			<!-- don't validate on release, otherwise the automatic db update won't work
```

Comment:
We shoud also look into `<property name="hibernate.cache.use_query_cache">true</property>` as mentioned here: https://www.ehcache.org/documentation/2.8/integrations/hibernate.html

Reply:
@BarnaBartha 
I would skip this in this PR, to make measuring easier. Could you create a follow up ticket for the query cache such that we can time this feature separately?

Reply:
The query cache makes sense only together with the 2nd level cache. To explain.. the query cache stores id's and then get all those objects form the 2nd level cache to be able to create the needed data for the query. That said we could activate query cache for infrastructure and user related queries, but do we have a performance problem there ?
We can create a ticket to investigate this but to activate it everywhere would only make sense if we would use 2nd level cache everywhere.

Reply:
https://github.com/hzi-braunschweig/SORMAS-Project/issues/10935

Reply:
I see, lets continue the discussion in the other ticket then! :)